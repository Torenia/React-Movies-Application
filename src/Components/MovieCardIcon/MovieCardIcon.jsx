import styled from 'styled-components';

export const MovieCardIcon = styled.button`
  position: absolute;
  top: 15px;
  right: 15px;
  font-size: 30px;
  background: #232323;
  width: 35px;
  height: 35px;
  border: none;
  border-radius: 50%;
  outline:none;
  cursor: pointer;
  
  &:after {
    content: "\\22EE";
    display: inline-block;
    color: #fff;
  }
`;


