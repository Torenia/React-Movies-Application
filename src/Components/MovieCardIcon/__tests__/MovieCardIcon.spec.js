import React from 'react';

import { MovieCardIcon } from '../MovieCardIcon';

describe('<MovieCardIcon />', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<MovieCardIcon />);
  });

  it('should render a movie card icon', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
