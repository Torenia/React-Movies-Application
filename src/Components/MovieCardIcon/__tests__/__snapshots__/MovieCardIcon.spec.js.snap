// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieCardIcon /> should render a movie card icon 1`] = `
.c0 {
  position: absolute;
  top: 15px;
  right: 15px;
  font-size: 30px;
  background: #232323;
  width: 35px;
  height: 35px;
  border: none;
  border-radius: 50%;
  outline: none;
  cursor: pointer;
}

.c0:after {
  content: "\\22EE";
  display: inline-block;
  color: #fff;
}

<button
  className="c0"
/>
`;
